performance-tests/pom.xml

```diff
@@ -104,6 +104,12 @@
         <plugins>
 
             <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>${jacoco.plugin.version}</version>
+            </plugin>
```

Comment:
To suppress it on performance tests and acceptance tests I will have to overload the surefire plugin config for these two modules. Since, surefire plugin requires argLine, if JaCoCo is not enabled the build fails since it does not find argLine. This does not slow down the build.
I can either overload it for these two modules or keep it as is with JaCoCo running. Please let me know what is more desirable.